### PR TITLE
[EAM-934] Show Events tab of extended in Work Orders of Equipment

### PR DESCRIPTION
**Pull request by pedroamarocern**
Note: the size at which the `EntityRegions` component uses 2 columns instead of 1 was changed from 960px to 1280px, to be able to fit the Events table in a readable way.

### CODE REVIEW

File: src/ui/pages/equipment/components/EquipmentWorkOrders.js [hunk 1]
```diff
@@ -32,20 +33,51 @@ const WO_FILTERS = {
...
+    let headers = ['Work Order', 'Equipment', 'Description', 'Status', 'Creation Date'];
+    let propCodes = ['number', 'object','desc', 'status', 'createdDate'];
+
+    if (workOrderFilter === WO_FILTER_TYPES.THIS) {
+        headers = ['Work Order', 'Description', 'Status', 'Creation Date'];
+        propCodes = ['number', 'desc', 'status', 'createdDate'];
+    }
+
+    // make spaces in header strings non-breaking so headers do not wrap to multiple lines
+    headers = headers.map(string => string.replaceAll(' ', '\u00a0'));
```
Comment (skamv) [inline src/ui/pages/equipment/components/EquipmentWorkOrders.js:60]: Ideally, this should be achieved with CSS, but I understand that it involves developments in eam-components

File: src/ui/components/entityregions/EntityRegions.js [hunk 2]
```diff
@@ -49,7 +49,7 @@ const EntityRegions = (props) => {
     const gridDimensions = {
         xs: 12,
         sm: 12,
-        md: showEqpTree || regionMaximized ? 12 : 12 / Object.keys(columns).length,
+        md: 12,
```
Comment (skamv) [inline src/ui/components/entityregions/EntityRegions.js:52]: Why was this changed?

Comment (pedroamarocern) [reply src/ui/components/entityregions/EntityRegions.js:52]: This was changed to be able to fit the table cleanly at all resolutions.

**Review by skamv — APPROVED** [review]
